<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="playButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="introButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>180</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="words">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">故事背景</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公元2022年，</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>神圣王子·芦苇于黑石城堡</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>击败了邪恶巨龙多拉贡，</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>55年后，也就是2077年</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>多拉贡的部下，北方军团首领</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北极熊韦天和其部下来势汹汹。</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>亲爱的勇士，</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>请守护时空之门吧！！！</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
